docs/modules/ROOT/pages/quickstart.adoc

```diff
@@ -1,6 +1,11 @@
 = Quickstart
+:navtitle: Quickstart
+:page-layout: tutorial
+:page-icon-role: bg-[var(--ds-neutral-900)]
+:page-toclevels: 1
+:keywords: Machine Learning Frameworks, Embedding Services, Data Warehouses, SDK
+:page-colab-link: https://colab.research.google.com/github/datastax/ragstack-ai/blob/main/examples/notebooks/quickstart.ipynb
```

Comment:
I just Viewed the file and don't see a colab link through that - do you also need to add
```
link:{page-colab-link}[Open in Colab]
```

Reply:
There's now a helper that will render the colab link and icon 
[https://github.com/riptano/docs-ui/blob/main/src/partials/tutorial-info.hbs](tutorial-info.hbs)